test/Microsoft.AspNetCore.Server.IISIntegration.Tests/IISMiddlewareTests.cs

```diff
@@ -191,7 +201,7 @@ public void PathBaseHiddenFromServer()
                 {
                     app.Run(context =>
                     {
-                        var auth = context.Features.Get<IHttpAuthenticationFeature>();
+                        var auth = context.RequestServices.GetService<IAuthenticationSchemeProvider>();
```

Comment:
not equivalent. You should still add the auth service in this case, it's testing the ForwardWindowsAuthentication flag. You may clone this test to make sure everything also works without the auth service.